daprdocs/content/en/operations/components/certification-lifecycle.md

```diff
@@ -0,0 +1,95 @@
+- compliant with the defined Dapr interfaces 
+- functionally correct and robust
+- well documented and maintained
+
+To make sure a component conforms to the standards set by Dapr, there are a set of tests run against a component in a Dapr maintainers managed environment. Once the tests pass consistently, the maturity level can be determined for a component. 
+
+## Certification levels 
+
+The levels are as follows:
+- [Alpha](#alpha)
+- [Beta](#beta)
+- [General availability (GA)](#general-availability-ga)
+
+### Alpha
+
+- The component implements the required interface and works as described in the specification
+- The component has documentation
+- The component might be buggy or might expose bugs on integration
+- The component may not pass all conformance tests
+- The component may not have conformance tests
+- Recommended for only non-business-critical uses because of potential for incompatible changes in subsequent releases
+
+All components start at the Alpha stage.
+
+### Beta
+
+- The component must pass all the component conformance tests defined to satisfy the component specification
+- The component conformance tests have been run in a Dapr maintainers managed environment
+- The component contains a record of the conformance test result reviewed and approved by Dapr maintainers with specific components-contrib version
+- Recommended for only non-business-critical uses because of potential for incompatible changes in subsequent releases
+
+### General Availability (GA)
+
+- Has at least two different users using the component in production
+- A GA component has a maintainer in the Dapr community or the Dapr maintainers
+- The component is well documented, tested and maintained across multiple versions of components-contrib repo
+
+## Conformance tests 
+
+Each component in the [components-contrib](https://github.com/dapr/components-contrib) repository needs to adhere to a set of interfaces defined by Dapr. Conformance tests are tests that are run on these component definitions with their associated backing services such that the component is tested to be conformant with the Dapr interface specifications and behavior. 
+
+The conformance tests are defined for the following building blocks: 
+
+- State store
+- Secret store
+- Bindings 
+- Pub/Sub
+
+To understand more about them see the readme [here](https://github.com/dapr/components-contrib/blob/master/tests/conformance/README.md).
+
+### Test requirements
+
+- The tests should validate the functional behavior and robustness of component based on the component specification
+- All the details needed to reproduce the tests are added as part of the component conformance test documentation
+
+## Component certification process
+
+### New component certification: Alpha->Beta or Beta->GA
+
+For a new component a request for component certification follows these steps. This applies for either certifying a component going from Beta from Alpha or GA from Beta.
```

Comment:
For a new component, a request for component ...

Reply:
Can you rephrase this? "This applies for either certifying a component going from Beta from Alpha or GA from Beta."